backend/server.py

```
@@ -1604,6 +1604,16 @@ async def _deferred_startup_cleanup() -> None:
     except Exception as exc:  # pragma: no cover - defensive
         log.warning("Boot purge scheduling failed (non-fatal): %s", exc)
 
+    # Start the self-heal background loop (runs every 5 min).
+    # Deduplicates tasks, resets brain failover deadlocks, clears stuck
+    # tasks, and clears Telegram webhook conflicts — all automatically.
+    try:
+        from services.self_heal import start_self_heal_scheduler
+        start_self_heal_scheduler()
+        log.info("Self-heal scheduler started")
+    except Exception as exc:  # pragma: no cover - defensive
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)